app/models.py

```diff
@@ -1414,3 +1414,24 @@ def serialize(self):
             'created_at': self.created_at.strftime(DATETIME_FORMAT),
             'updated_at': self.updated_at.strftime(DATETIME_FORMAT) if self.updated_at else None
         }
+
+
+class NotificationSmsSender(db.Model):
```

Comment:
Should probably be NotificationEmailReplyTo?

Reply:
Yes updated